CHANGELOG.md

```diff
@@ -27,9 +27,12 @@ Each issue fixed should contain one bullet summarizing the work done.
 <hr />
 
 ## [Unreleased]
+* [CIO-259](https://github.com/jwu910/check-it-out/issues/259)
+User can now terminate loading states with escape hotkeys (q, esc, C-c)
```

Comment:
Can we move this into one line? I want to keep the log consistent.

Reply:
@9reeno

Reply:
@jwu910 Sorry, thought I got that. pushing soon.

Reply:
No worries! Almost done!